notebooks/page view functional network/page_view_approach_economic_recovery.ipynb

```diff
@@ -0,0 +1,1507 @@
+    "# which pages are in the economic recovery list but not in the subgraph list? \n",
+    "getPagesNotInSubGraph(g)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Equal to or lower than 30 sessions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# pages with less than 30 session hits \n",
+    "remove = [node for node, session in g.nodes(data=True) if session['sessionHitsAll'] <= 30]\n",
+    "\n",
+    "# remove nodes with less than 30 session hits \n",
+    "g.remove_nodes_from(remove)\n",
+    "\n",
+    "# how many nodes and edges\n",
+    "nx.info(g)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# how many of the start a business pages in the functional graph?\n",
+    "getNoOfTruePages(g)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# which pages are in the economic recovery list but not in the subgraph list? \n",
+    "getPagesNotInSubGraph(g)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Remove pages with max shortest page paths from `seed0` pages >= 3"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Identify the shortest page paths from seed0 pages to all other pages. \n",
+    "\n",
+    "Frequency distribution of shortest page path lengths: \n",
+    "- `seed0.1`; /browse/working/finding-job; {4: 2, 3: 137, 2: 1272, 1: 138, 0: 1}\n",
+    "- `seed0.2`: /topic/further-education-skills/apprenticeships; {4: 5, 3: 392, 2: 1047, 1: 105, 0: 1}\n",
+    "\n",
+    "\n",
+    "If the shortest page path is equal to or greater than 3, remove from the nodes from the subgraph:\n",
+    "- 1521 nodes\n",
+    "- 37/41 \n",
+    "- Pages in the manually defined list not in the subgraph: [`/guidance/international-trade-products-and-schemes`,\n",
+    " `/guidance/recovery-loan-scheme`, `/find-driving-instructor-training`, `/how-to-claim-universal-credit`, `/government/collections/financial-support-for-businesses-during-coronavirus-covid-19`, `/agricultural-skills-and-training`, `/education`]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# sort nodes by closeness to seed 0 pages\n",
+    "shortest_paths_seed0_1 = nx.shortest_path_length(g, source='/browse/working/finding-job')\n",
+    "shortest_paths_seed0_2 = nx.shortest_path_length(g, source='/topic/further-education-skills/apprenticeships')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# sort nodes in descending order\n",
+    "finding_job={k: v for k, v in sorted(shortest_paths_seed0_1.items(), key=lambda item: item[1], reverse=True)}"
```

Comment:
Very nice dict comprehension. Not worth rewriting this, but just a tip to store in your mind:
`finding_job = {k: v for k, v in sorted(shortest_paths_seed0_1.items(), key = lambda item: -item[1])}` is equivalent to this code. Notice the `reverse=True` is removed and a minus sign comes before `item[1]`.